lib/kennel/optional_validations.rb

```
@@ -1,11 +1,15 @@
 # frozen_string_literal: true
 module Kennel
   module OptionalValidations
-    ValidationMessage = Struct.new(:text)
+    ValidationMessage = Struct.new(:tag, :text)
+
+    UnignorableTag = Struct.new(:inspect) # rubocop:disable Lint/StructNewOverride
+    NO_TAG_CHOSEN = UnignorableTag.new("<no tag chosen>")
+    NO_TAG = UnignorableTag.new("<not ignorable>")
 
     def self.included(base)
-      base.settings :validate
-      base.defaults(validate: -> { true })
+      base.settings :ignored_errors, :validate
+      base.defaults(ignored_errors: -> { [] }, validate: true)
```

should we call this `ignored_validations` ?

---

If you like. That's a good name.

---

Welll.... we don't actually ignore the validation, we still run the validation and then ignore the error.

But it's not something I have a strong opinion on. Some combination of "ignore/skip/expect/suppress"\[ed\] "errors/validations/validation_errors" works.